arm/Microsoft.DomainServices/AzureADDomainServices/readme.md

```diff
@@ -0,0 +1,160 @@
+# Azure Active Directory Domain Services `[Microsoft.DomainServices/AzureADDomainServices]`
```

Comment:
Once any changes to the `deploy.bicep` template are implemented, please re-run the `Set-ModuleReadMe` script to refresh the this file

Reply:
@MrMCake my apologies. Where exactly can I find this script?

Reply:
Hey @mararam , the script can be found in path [utilities/tools/Set-ModuleReadMe.ps1](https://github.com/Azure/ResourceModules/blob/main/utilities/tools/Set-ModuleReadMe.ps1)

Reply:
@MrMCake I ran this now after committing the latest changes, but I haven't seen any output/updates after running it. 

Tool binary path>> .\Set-ModuleReadMe.ps1 -TemplateFilePath 'aadds bicep path' -ReadMeFilePath 'aadds read me file'

Reply:
The `Set-ModuleReadme` script contains actually multiple functions. After loading the script (and the contained function) you can invoke the Update by invoking `Set-ModuleReadme` explicitely:
```PowerShell
. 'Set-ModuleReadMe.ps1'
Set-ModuleReadMe -TemplateFilePath '<PathToTheDeployBicep>'
```

Reply:
@MrMCake Thank you the command. I have made some corrections in the main.bicep now so I could able to run and generate the readme.md file successfully. Please review and let me know. Thanks.